apps/dotcom/client/src/components/StoreErrorScreen.tsx

```diff
@@ -47,6 +47,11 @@ export function StoreErrorScreen({ error }: { error: Error }) {
 				message = 'The room you are trying to connect to does not exist.'
 				break
 			}
+			case TLIncompatibilityReason.NotAuthorized: {
+				header = 'Not authorized'
+				message = 'You are not authorized to view this room.'
```

Comment:
Nitpick on language: "You need to be authorized to view this room." (missing authentication in this case, not authentication that's forbidden)

Reply:
actually, also it should be 'Unauthorized' instead of 'Not authorized' for the same reason.

Reply:
good suggestions